maro/rl/learning/policy_manager.py

```diff
@@ -356,3 +380,20 @@ def policy_host(
             }
             logger.debug(f"total policy update time: {time.time() - t0}")
             proxy.reply(msg, tag=MsgTag.LEARN_DONE, body=msg_body)
+        elif msg.tag == MsgTag.COMPUTE_GRAD:
+            t0 = time.time()
+            msg_body = {MsgKey.LOSS_INFO: dict()}
+            for name, batch in msg.body[MsgKey.GRAD_TASK].items():
+                if MsgKey.POLICY_STATE in msg.body:
```

Comment:
It will learn the batch of data with local policy parameters instead of latest parameters in the message.

Reply:
Since the `grad_worker` is stateless, in which case will we send a message without POLICY_STATE?

Reply:
One case maybe: only one grad worker for a policy, so that sending `POLICY_STATE` is unnecessary.
However, that might be trivial, can be removed for simplicity.🤣